client/shipping-label/state/test/reducer.js

```diff
@@ -0,0 +1,245 @@
+import reducer from '../reducer';
+import {
+	moveItem,
+	addPackage,
+	removePackage,
+	setPackageType,
+	savePackages,
+} from '../actions';
+import hoek from 'hoek';
+
+const initialState = {
+	form: {
+		packages: {
+			all: {
+				customPackage1: {
+					inner_dimensions: '1 x 2 x 3',
+					box_weight: 3,
+				},
+				customPackage2: {},
+			},
+			selected: {
+				weight_0_custom1: {
+					items: [
+						{
+							product_id: 123,
+							weight: 1,
+						},
+					],
+				},
+				weight_1_custom1: {
+					items: [
+						{
+							product_id: 456,
+							weight: 2,
+						},
+					],
+				},
+			},
+			unpacked: [],
+			isPacked: true,
+		},
+	},
+	openedPackageId: 'weight_0_custom1',
+};
+
+describe( 'Label purchase form reducer', () => {
+	afterEach( () => {
+		// make sure the state hasn't been mutated
+		// after each test
+		expect( initialState ).to.eql( {
```

Comment:
Could using [`deepFreeze()`](https://github.com/substack/deep-freeze) on `initialState` be a way to avoid this?